### DISCUSSION TITLE
need help!

### DISCUSSION THREAD

**Discussion by carrionfs** [xxyzz/WordDumb#148] [General]
When I tried to change the set of the gloss language and the lemma language, it showed me this:
calibre, version 6.26.0
错误： 未处理的异常: <b>AttributeError</b>:'SA' object has no attribute 'job_manager'

calibre 6.26  embedded-python: True
Windows-10-10.0.19041-SP0 Windows ('64bit', 'WindowsPE')
('Windows', '10', '10.0.19041')
Python 3.10.1
Windows: ('10', '10.0.19041', 'SP0', 'Multiprocessor Free')
Interface language: zh_CN
Successfully initialized third party plugins: Chinese Text Conversion (3, 0, 1) && WordDumb (3, 29, 5)
Traceback (most recent call last):
  File "calibre_plugins.worddumb.config", line 255, in open_choose_lemma_lang_dialog
  File "calibre_plugins.worddumb.config", line 313, in run_threaded_job
AttributeError: 'SA' object has no attribute 'job_manager'

 and I don't know how to fix it.
Besides, this plugin worked yesterday but it gave wrong meanings for every words. When I pressed the word it showed another word and its meaning, and I don't know what's wrong with it.

**Comment by xxyzz** [comment]
This error was fixed in the master branch: https://github.com/xxyzz/WordDumb/commit/817c193e52b512c94274d27cd3f5154375118e1a

Have you enabled the "Use Wiktionary definition" optional? If it's enabled, word wise language must be changed to Chinese on Kindle to use Wiktionary word wise database file.
